src/harness/fake_sas.py

```diff
@@ -247,6 +247,9 @@ def TriggerPpaCreation(self, request, ssl_cert=None, ssl_key=None):
     return 'zone/ppa/fake_sas/%s/%s' % (request['palIds'][0]['palId'],
                                         uuid.uuid4().hex)
 
+  def GetFullActivityDump(self):
```

Comment:
I think you need to move this to class FakeSas and instead implement TriggerFullActivityDump here.